### PR TITLE
ci: deploy storybook to github pages automatically

### PR DESCRIPTION
**Pull request by ZengJunyong**
Once PR is merged to master, https://rebilly.github.io/rebilly-recomponents is updated automatically.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,10 +32,18 @@ jobs:
         - yarn lint
         - yarn test
     - stage: deploy
-      name: 'Release recomponents to NPM'
+      name: 'Release recomponents to NPM and deploy storybook to github pages'
       before_script:
         - cd packages/recomponents
       if: branch = master
```
Comment (ZengJunyong) [inline .travis.yml]: @followyouhome Just curious why I pushed a PR, the script is executed? so what does `if: branch = master` mean here?

Comment (followyouhome) [reply .travis.yml]: to prevent duplicated builds - w/o this condition we will have build for each branch + build for PR

Comment (ZengJunyong) [reply .travis.yml]: Agreed. With this condition, it looks like when PR is merged to master, the deploy will be executed, right? But I noticed when I created a PR, it is executed as well...

**Comment by followyouhome** [comment]
Please don't use tickets in branch names or PR titles

File: .travis.yml (hunk 1 above)
Comment (mathewsonke) [reply .travis.yml]: are you sure it was the deploy script that executed? This file is still configured to `build`, `lint`, and `test` on all PRs

File: .travis.yml [hunk 2]
```diff
@@ -33,10 +32,18 @@ jobs:
...
       script:
         - yarn build
-        - yarn semantic-release
\ No newline at end of file
+        - yarn semantic-release
+        - yarn build-storybook
+        - cd .out
+        - git init
+        - git config user.name "junyong"
```
Comment (mathewsonke) [inline .travis.yml]: We can't replace this with more generic credentials, can we?

**Comment by mathewsonke** [comment]
> Please don't use tickets in branch names or PR titles

Is this because of open source?

File: .travis.yml (hunk 1 above)
Comment (ZengJunyong) [reply .travis.yml]: Let's first agree with below point?
1) if master: build, deploy storybook, semantic-release
2) if other branches(PR): lint, test
Then let's adjust the `.travis.yml` file.

File: .travis.yml (hunk 2 above)
Comment (ZengJunyong) [reply .travis.yml]: I think Can, need a generic GitHub account maybe. 
BTW: I'm using my Github account to `git push` to `gh-pages` branch now.

Comment (ZengJunyong) [reply .travis.yml]: Oh for the `user.name`, I think whatever it is, it should be OK

**Comment by followyouhome** [comment]
Yes, I was asked no to do that earlier

**Comment by ZengJunyong** [comment]
> Yes, I was asked no to do that earlier

Got it

**Comment by ZengJunyong** [comment]
OK, if we need this in the future, just copy the related script.

**Pull request closed without merging**